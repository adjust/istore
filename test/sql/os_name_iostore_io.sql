BEGIN;
CREATE TABLE os_name_istore_io AS SELECT 'android=>1,android=>2'::os_name_istore;
SELECT * FROM os_name_istore_io;
ROLLBACK;

BEGIN;
CREATE TABLE os_name_istore_io AS SELECT '"android"=>"1","android"=>"2"'::os_name_istore;
SELECT * FROM os_name_istore_io;
ROLLBACK;

BEGIN;
CREATE TABLE os_name_istore_io AS SELECT '"android"=>"1","ios"=>"2"'::os_name_istore;
SELECT * FROM os_name_istore_io;
ROLLBACK;

BEGIN;
CREATE TABLE os_name_istore_io AS SELECT '"ios"=>"+1","android"=>"2"'::os_name_istore;
SELECT * FROM os_name_istore_io;
ROLLBACK;

BEGIN;
CREATE TABLE os_name_istore_io AS SELECT ' "ios"=>"+1","android"=>"2"'::os_name_istore;
SELECT * FROM os_name_istore_io;
ROLLBACK;

BEGIN;
CREATE TABLE os_name_istore_io AS SELECT '"ios"=> "+1","android"=>"2"'::os_name_istore;
SELECT * FROM os_name_istore_io;
ROLLBACK;

BEGIN;
CREATE TABLE os_name_istore_io AS SELECT '"ios"=>"+1" ,"android"=>"2"'::os_name_istore;
SELECT * FROM os_name_istore_io;
ROLLBACK;

BEGIN;
CREATE TABLE os_name_istore_io AS SELECT '"ios"=>"+1", "android"=>"2"'::os_name_istore;
SELECT * FROM os_name_istore_io;
ROLLBACK;

BEGIN;
CREATE TABLE os_name_istore_io AS SELECT '"ios"=>"+1","android"=>"2" '::os_name_istore;
SELECT * FROM os_name_istore_io;
ROLLBACK;

BEGIN;
CREATE TABLE os_name_istore_io AS SELECT 'ios=>+1,android=>2 '::os_name_istore;
SELECT * FROM os_name_istore_io;
ROLLBACK;
