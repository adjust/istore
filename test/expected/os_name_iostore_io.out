BEGIN;
CREATE TABLE os_name_istore_io AS SELECT 'android=>1,android=>2'::os_name_istore;
SELECT * FROM os_name_istore_io;
 os_name_istore 
----------------
 "android"=>"3"
(1 row)

ROLLBACK;
BEGIN;
CREATE TABLE os_name_istore_io AS SELECT '"android"=>"1","android"=>"2"'::os_name_istore;
SELECT * FROM os_name_istore_io;
 os_name_istore 
----------------
 "android"=>"3"
(1 row)

ROLLBACK;
BEGIN;
CREATE TABLE os_name_istore_io AS SELECT '"android"=>"1","ios"=>"2"'::os_name_istore;
SELECT * FROM os_name_istore_io;
      os_name_istore       
---------------------------
 "android"=>"1","ios"=>"2"
(1 row)

ROLLBACK;
BEGIN;
CREATE TABLE os_name_istore_io AS SELECT '"ios"=>"+1","android"=>"2"'::os_name_istore;
SELECT * FROM os_name_istore_io;
      os_name_istore       
---------------------------
 "android"=>"2","ios"=>"1"
(1 row)

ROLLBACK;
BEGIN;
CREATE TABLE os_name_istore_io AS SELECT ' "ios"=>"+1","android"=>"2"'::os_name_istore;
SELECT * FROM os_name_istore_io;
      os_name_istore       
---------------------------
 "android"=>"2","ios"=>"1"
(1 row)

ROLLBACK;
BEGIN;
CREATE TABLE os_name_istore_io AS SELECT '"ios"=> "+1","android"=>"2"'::os_name_istore;
SELECT * FROM os_name_istore_io;
      os_name_istore       
---------------------------
 "android"=>"2","ios"=>"1"
(1 row)

ROLLBACK;
BEGIN;
CREATE TABLE os_name_istore_io AS SELECT '"ios"=>"+1" ,"android"=>"2"'::os_name_istore;
SELECT * FROM os_name_istore_io;
      os_name_istore       
---------------------------
 "android"=>"2","ios"=>"1"
(1 row)

ROLLBACK;
BEGIN;
CREATE TABLE os_name_istore_io AS SELECT '"ios"=>"+1", "android"=>"2"'::os_name_istore;
SELECT * FROM os_name_istore_io;
      os_name_istore       
---------------------------
 "android"=>"2","ios"=>"1"
(1 row)

ROLLBACK;
BEGIN;
CREATE TABLE os_name_istore_io AS SELECT '"ios"=>"+1","android"=>"2" '::os_name_istore;
SELECT * FROM os_name_istore_io;
      os_name_istore       
---------------------------
 "android"=>"2","ios"=>"1"
(1 row)

ROLLBACK;
BEGIN;
CREATE TABLE os_name_istore_io AS SELECT 'ios=>+1,android=>2 '::os_name_istore;
SELECT * FROM os_name_istore_io;
      os_name_istore       
---------------------------
 "android"=>"2","ios"=>"1"
(1 row)

ROLLBACK;
